documentation/modules/exploit/windows/http/meddream_pacs_rce.md

```diff
@@ -0,0 +1,55 @@
+## Details
+This module exploits a vulnerability found in MedDream PacsServer 6.8.3.751, which allows unauthenticated attackers to perform Remote Code Execution via an arbitrary PHP file upload. Depending on how the web server is configured to run will determine what level of access privilege obtained.
```

Comment:
up to and including this version number?  Believed to be exploitable in versions before? It didn't seem clear from the EDB modules.